src/Symfony/Component/Form/composer.json

```diff
@@ -33,7 +33,10 @@
         "symfony/http-foundation": ""
     },
     "autoload": {
-        "psr-0": { "Symfony\\Component\\Form\\": "" }
+        "psr-0": { "Symfony\\Component\\Form\\": "" },
+        "exclude-from-classmap": [
+            "/Tests/"
```

Comment:
This folder, just like the Test/ folder in the FrameworkBundle, provides test helpers to use in your own unit/functional tests